File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        new=[]
        for i in range(len(nums)):
            for j in range(len(nums)):
                if i!=j:
                    if nums[i]+nums[j]==target:
                        new.append(i)
        return new

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int r=nums.length-1,l=0;
        while(l<r){
            if(nums[r]+nums[l]==target){
                return new int[]{l,r};
            }
            r--;
            if(r==l){
                l++;
                r=nums.length-1;
            }
        }
        return new int[]{-1,1};
    }
}